ci/ci/ci.py

```diff
@@ -207,28 +209,38 @@ async def get_job(request, userdata):
     return await render_template('ci', request, userdata, 'job.html', page_context)
 
 
+def filter_wbs(wbs, pred):
+    return [{**wb, 'prs': [pr for pr in wb['prs'] if pred(pr)]} for wb in wbs]
+
+
 @routes.get('/me')
 @web_authenticated_developers_only()
 async def get_user(request, userdata):
     username = userdata['username']
+    gh_username = None
+    pr_wbs = []
+    review_wbs = []
     for user in AUTHORIZED_USERS:
         if user.hail_username == username:
             gh_username = user.gh_username
-            wbs = [
-                await watched_branch_config(request.app, wb, i, pr_author=gh_username)
-                for i, wb in enumerate(watched_branches)
-            ]
+            wbs = [await watched_branch_config(request.app, wb, i) for i, wb in enumerate(watched_branches)]
+            pr_wbs = filter_wbs(wbs, lambda pr: gh_username == pr['author'])
+            review_wbs = filter_wbs(wbs, lambda pr: gh_username in pr['assignees'])
             break
-    else:
-        gh_username = None
-        wbs = []
     batch_client = request.app['batch_client']
     dev_deploys = batch_client.list_batches(f'user={username} dev_deploy=1', limit=10)
     dev_deploys = sorted([b async for b in dev_deploys], key=lambda b: b.id, reverse=True)
+
+    team_random_member = {
+        team: random.choice([user for user in AUTHORIZED_USERS if team in user.teams]).gh_username for team in TEAMS
+    }
+
     page_context = {
         'username': username,
         'gh_username': gh_username,
-        'watched_branches': wbs,
+        'pr_wbs': pr_wbs,
+        'review_wbs': review_wbs,
```

Comment:
So this is great, but scorecard does something different. Scorecard tells you things that you need to take action on. That means:
- your PRs that are failing tests
- your PRs that have changes requested reviews
- PRs assigned to you that you have no extant review (e.g. it got dismissed, you haven't reviewed yet)

Reply:
Ya this does take a different approach, but I think the information is still there. For any PR, it shows the build state and review state, so you can see which PRs are failing tests and which PRs have review pending / changes requested / approved for your PRs as well as those assigned to you.

Reply:
Or do you mean that it would be nice to eliminate PRs which don't require action?